File: main/java/com/example/appmovil/model/Usuario.java
package com.example.appmovil.model;

public class Usuario {
    private String user;
    private String contrasenia;
    private String nombre;
    private String apellido;
    private String correo;
    private String tipoUsuario;

    public Usuario(String user, String contra, String nombre, String apellido, String correo,String tipoUsuario)
    {
        this.user = user;
        this.contrasenia = contra;
        this.nombre = nombre;
        this.apellido = apellido;
        this.correo = correo;
        this.tipoUsuario=tipoUsuario;
    }
    public Usuario(String user, String contra, String nombre, String apellido, String correo)
    {
        this.user = user;
        this.contrasenia = contra;
        this.nombre = nombre;
        this.apellido = apellido;
        this.correo = correo;

    }

    public Usuario()
    {
    }

    public String getUser() {
        return user;
    }

    public void setUser(String user) {
        this.user = user;
    }

    public String getContrasenia() {
        return contrasenia;
    }

    public void setContrasenia(String contrasenia) {
        this.contrasenia = contrasenia;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getApellido() {
        return apellido;
    }

    public void setApellido(String apellido) {
        this.apellido = apellido;
    }

    public String getCorreo() {
        return correo;
    }

    public void setCorreo(String correo) {
        this.correo = correo;
    }

    public String getTipoUsuario() {
        return tipoUsuario;
    }

    public void setTipoUsuario(String tipoUsuario) {
        this.tipoUsuario = tipoUsuario;
    }
}

File: BackendPostgres/BackendPostgres/Models/Usuario.cs
namespace BackendPostgres.Models
{
    public class Usuario
    {
        private string user;
        private string contrasenia;
        private string nombre;
        private string apellido;
        private string correo;
        private string tipoUsuario;

        public Usuario(string user, string contra, string nombre, string apellido, string correo)
        {
            this.user = user;
            this.contrasenia = contra;
            this.nombre = nombre;
            this.apellido = apellido;
            this.correo = correo;
        }

        public Usuario()
        {
        }

        public Usuario(string user, string contrasenia, string nombre, string apellido, string correo, string tipoUsuario)
        {
            this.user = user;
            this.contrasenia = contrasenia;
            this.nombre = nombre;
            this.apellido = apellido;
            this.correo = correo;
            this.tipoUsuario = tipoUsuario;
        }

        public string User { get => user; set => user = value; }
        public string Contrasenia { get => contrasenia; set => contrasenia = value; }
        public string Nombre { get => nombre; set => nombre = value; }
        public string Apellido { get => apellido; set => apellido = value; }
        public string TipoUsuario { get => tipoUsuario; set => tipoUsuario = value; }
        public string Correo { get => correo; set => correo = value; }
    }
}
